ci/Dockerfile.coverity

```diff
@@ -21,29 +21,43 @@ COPY ./cov-analysis-linux64-2022.12.2 /cov
 
 RUN apt-get remove --yes cmake && apt-get update && apt-get install -y rapidjson-dev && pip3 install cmake
 
-WORKDIR /example_cpp_client/cpp
-RUN bazel build @org_tensorflow//tensorflow/core:framework
+#WORKDIR /example_cpp_client/cpp
+#RUN bazel build @org_tensorflow//tensorflow/core:framework
 
 WORKDIR /ovms/
 RUN /cov/bin/cov-configure --gcc --config coverity_config.xml && \
     /cov/bin/cov-configure --comptype gcc --compiler /usr/bin/gcc && \
     /cov/bin/cov-build --dir cov-int bash -c 'bazel shutdown; bazel clean; bazel build --spawn_strategy=standalone //src:static_analysis && cmake /client/cpp/kserve-api && make --jobs=$(nproc) && \
     cd /ovms/src/custom_nodes/tokenizer && \
     mkdir -p build && cd build && \
-    cmake .. && make --jobs=$(nproc) && \
-    cd /example_cpp_client/cpp && \
-    bazel build --spawn_strategy=standalone //src:all'
+    cmake .. && make --jobs=$(nproc)'
+#    cd /example_cpp_client/cpp && \
```

Comment:
Why skipping cpp clients?

Reply:
not compatible with the current TFS dependency version in the build image